models/sandbox_route_group_endpoint.go

```diff
@@ -0,0 +1,53 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+)
+
+// SandboxRouteGroupEndpoint sandbox route group endpoint
+//
+// swagger:model sandbox.RouteGroupEndpoint
+type SandboxRouteGroupEndpoint struct {
```

Comment:
This model seems a bit odd - how did it derive this name?

Reply:
From this? https://github.com/signadot/signadot/blob/master/workspaces/app/api/signadotv2/routegroups/endpoint.go#L30

Reply:
I think we should change that and regenerate. It should be `routegroup.Endpoint`

Reply:
Another case that looks a bit off is https://github.com/signadot/signadot/blob/f02b1be6de2e0219d21529ccc6162e7e6174a534/workspaces/app/docs/dist/swagger.json#L215

Reply:
There's a tricky issue here that I ran into prior. Since we have 2 endpoint fields, one at the top level, and another under spec now, we should name them differently to prevent auto-generated clients from getting confused. I'd propose `routegroup.endpoint` (for top-level field) and `routegroup.spec.endpoint` for the one within spec.

Reply:
/xref https://github.com/signadot/signadot/pull/2682